mono/dodge_the_creeps/Player.cs

```diff
@@ -3,86 +3,67 @@
 public partial class Player : Area2D
 {
     [Signal]
-    public delegate void Hit();
+    public delegate void HitEventHandler();
 
     [Export]
-    public int speed = 400; // How fast the player will move (pixels/sec).
+    public int Speed = 400; // How fast the player will move (pixels/sec).
```

Comment:
While you are changing this, would you mind making these public fields be properties instead?

```suggestion
    public int Speed { get; set; } = 400; // How fast the player will move (pixels/sec).
```